src/lib/server/r2.ts

```diff
@@ -0,0 +1,44 @@
+import { CreateBucketCommand, ListBucketsCommand, S3Client } from "@aws-sdk/client-s3";
+import {
+	CLOUDFLARE_ACCESS_KEY_ID,
+	CLOUDFLARE_R2_ENDPOINT,
+	CLOUDFLARE_SECRET_ACCESS_KEY,
+} from "./config";
+
+export const CLOUDFLARE_R2_BUCKET = "unicourse";
```

Comment:
I think it should be configurable since we may have multiple bucket for different stages which are all under the same endpoint.

Reply:
make sense. fixed.